lib/src/cta/608/findCta608Nalus.ts

```diff
@@ -14,7 +14,7 @@ export function findCta608Nalus(raw: DataView, startPos: number, size: number):
 	const cta608NaluRanges = [];
 
 	// Check SEI data according to ANSI-SCTE 128
-	const isCTA608SEI = function (payloadType: number, payloadSize: number, raw: DataView, pos: number) {
+	const isCTA608SEI = (payloadType: number, payloadSize: number, raw: DataView, pos: number) => {
```

Comment:
avoided mixed arrow function / declaration functions, and simply using arrow functions here